### PR TITLE
docs: combine bot and CLI configuration

### PR DESCRIPTION
**Pull request by tenshiAMD**
**What**:
- Fixes https://github.com/all-contributors/all-contributors/issues/629

**Why**:
- To make configuration clear for both bot and CLI

**How**:
- Update documentation for bot and CLI

**Checklist**:

- [x] Documentation
- [x] Ready to be merged 
- [x] Added myself to contributors table. 
[Bot Usage](https://allcontributors.org/docs/en/bot/installation#4-update-your-contributing-documentation)

### CODE REVIEW

File: docs/cli/configuration.md [hunk 1]
```diff
@@ -4,49 +4,4 @@ title: CLI Configuration
...
-    "custom": {
-      "symbol": "🔭",
-      "description": "A custom contribution type.",
-      "link": "[<%= symbol %>](<%= url %> \"<%= description %>\"),"
-    }
-  },
-  "contributors": []
-}
-```
+> Please refer to this [documentation for the configuration](../bot/configuration).
```
Comment (Roshanjossey) [inline docs/cli/configuration.md:7]:
```suggestion
> Please refer to this [documentation](../bot/configuration) for the configuration.
```

**Review by Berkmann18 — COMMENTED** [review]
It might be worth moving that out of `/bot/` or making it clear that it applies to the CLI as well.

**Comment by tenshiAMD** [comment]
> It might be worth moving that out of `/bot/` or making it clear that it applies to the CLI as well.

@Berkmann18 I already added a note and reference in the CLI documentation site, or do you want CLI to have it then bot documentation will only have reference?

**Comment by Berkmann18** [comment]
> > It might be worth moving that out of `/bot/` or making it clear that it applies to the CLI as well.
> 
> @Berkmann18 I already added a note and reference in the CLI documentation site, or do you want CLI to have it then bot documentation will only have reference?

Oh yeah, I missed that last line.

**Review by Berkmann18 — APPROVED** [review]

**Pull request merged**